src/app/components/selectable-box/double-column/DoubleSelectableBox.jsx

```diff
@@ -27,8 +27,8 @@ const DoubleSelectableBox = props => {
     const { sortedItems, requestSort, sortConfig } = useSort(props.items, activeSort);
 
     const handleSortClick = (key, direction) => {
-        setActiveSort({key, direction});
-        requestSort({key, direction});
+        setActiveSort({ key, direction });
+        requestSort({ key, direction });
```

Comment:
I wonder if we really need the `direction` param in `handleSortClick` as it seems to always be "ASC"?

Reply:
yes, I would like to leave it as it uses the useSort function and and would like it to be generic for reusability. no harm in sending this value